iot-agent/docker-compose/orion-ld.yml

```diff
@@ -0,0 +1,136 @@
+# WARNING: Do not deploy this tutorial configuration directly to a production environment
```

Comment:
[Major] What is the purpose of this file I think no need to this file here so please verify.